### ISSUE TITLE
Shuffle Training Data

### ISSUE THREAD

**Issue by znavidi** [FoundationVision/Infinity#102]
Hi authors, and thanks for your amazing contribution!

I need to shuffle the training data for training/fine-tuning, and looking into the code I found the following shuffle variable is set in the  dataset_t2i_iterable.py, which is set to True. Could you pls help me understand if that means the training data is shuffled each time at the beginning of training? if yes, is that based on input args.seed?

<img width="351" alt="Image" src="https://github.com/user-attachments/assets/908dc1e2-55f6-489b-a78b-55a88443be3b" />

greatly appreciate your confirmation.

**Comment by znavidi** [comment]
I learned that setting different seed values results in different batches as I monitored.
